src/prototypes/single-article/example/example.twig

```diff
@@ -0,0 +1,327 @@
+<div id="single-article">
+  {% include '@cloudfour/components/sky-nav/sky-nav.twig' with {
+    "class": "t-dark",
+    "include_main_demo": false,
+    "menu": {
+      "items": [
+        {
+          "link": "/does",
+          "title": "What We Do",
+          "current": true
+        },
+        {
+          "link": "/believes",
+          "title": "Our Approach"
+        },
+        {
+          "link": "/made",
+          "title": "Our Work"
+        },
+        {
+          "link": "/thinks",
+          "title": "Articles"
+        },
+        {
+          "link": "/presents",
+          "title": "Speaking"
+        },
+        {
+          "link": "/contributes",
+          "title": "Labs"
+        },
+        {
+          "link": "/is",
+          "title": "Team"
+        }
+      ]
+    }
+  } only %}
+  {% embed '@cloudfour/objects/container/container.twig' with {
+    class: 'o-container--prose o-container--pad main-container'} %}
+    {% block content %}
+      {% embed '@cloudfour/objects/rhythm/rhythm.twig' with {
+        class: 'o-rhythm--generous-headings' } %}
+        {% block content %}
+          {% include '@cloudfour/components/heading/heading.twig' with {
+            level: 0,
```

Comment:
Did you try `level: -1` out of curiosity? It looks a little nicer to me when it's larger.